cpp/examples/sde_seir2v.cpp

```diff
@@ -0,0 +1,96 @@
+
+    double t0   = 0.;
+    double tmid = 100.;
+    double tmax = 400.;
+    double dt   = 0.1;
+
+    double total_population = 1000;
+
+    mio::RandomNumberGenerator rng;
+    std::initializer_list<uint32_t> seeds = {14159265u, 35897932u};
+    rng.seed(seeds);
+
+    mio::log_info("Simulating SIR; t={} ... {} with dt = {}.", t0, tmax, dt);
+
+    mio::sseir2v::Model model;
+
+    model.populations[{mio::Index<mio::sseir2v::InfectionState>(mio::sseir2v::InfectionState::ExposedV1)}]  = 0;
+    model.populations[{mio::Index<mio::sseir2v::InfectionState>(mio::sseir2v::InfectionState::ExposedV2)}]  = 0;
+    model.populations[{mio::Index<mio::sseir2v::InfectionState>(mio::sseir2v::InfectionState::InfectedV1)}]  = 100;
+    model.populations[{mio::Index<mio::sseir2v::InfectionState>(mio::sseir2v::InfectionState::InfectedV2)}]  = 0.01;
+    model.populations[{mio::Index<mio::sseir2v::InfectionState>(mio::sseir2v::InfectionState::RecoveredV1)}] = 0;
+    model.populations[{mio::Index<mio::sseir2v::InfectionState>(mio::sseir2v::InfectionState::RecoveredV2)}] = 0;
+    model.populations[{mio::Index<mio::sseir2v::InfectionState>(mio::sseir2v::InfectionState::ExposedV1V2)}]  = 0;
+    model.populations[{mio::Index<mio::sseir2v::InfectionState>(mio::sseir2v::InfectionState::InfectedV1V2)}]  = 0;
+    model.populations[{mio::Index<mio::sseir2v::InfectionState>(mio::sseir2v::InfectionState::RecoveredV1V2)}] = 0;
+    model.populations[{mio::Index<mio::sseir2v::InfectionState>(mio::sseir2v::InfectionState::Susceptible)}] =
+        total_population -
+        model.populations[{mio::Index<mio::sseir2v::InfectionState>(mio::sseir2v::InfectionState::ExposedV1)}] -
+        model.populations[{mio::Index<mio::sseir2v::InfectionState>(mio::sseir2v::InfectionState::ExposedV2)}] -
+        model.populations[{mio::Index<mio::sseir2v::InfectionState>(mio::sseir2v::InfectionState::InfectedV1)}] -
+        model.populations[{mio::Index<mio::sseir2v::InfectionState>(mio::sseir2v::InfectionState::InfectedV2)}] -
+        model.populations[{mio::Index<mio::sseir2v::InfectionState>(mio::sseir2v::InfectionState::RecoveredV1)}] -
+        model.populations[{mio::Index<mio::sseir2v::InfectionState>(mio::sseir2v::InfectionState::RecoveredV2)}] -
+        model.populations[{mio::Index<mio::sseir2v::InfectionState>(mio::sseir2v::InfectionState::ExposedV1V2)}] -
+        model.populations[{mio::Index<mio::sseir2v::InfectionState>(mio::sseir2v::InfectionState::InfectedV1V2)}] -
+        model.populations[{mio::Index<mio::sseir2v::InfectionState>(mio::sseir2v::InfectionState::RecoveredV1V2)}];
+
+    double beta = mio::DistributionAdapter<std::lognormal_distribution<double>>::get_instance()(rng, std::log(0.08), 0.5);
+    double kappa = mio::DistributionAdapter<std::lognormal_distribution<double>>::get_instance()(rng, std::log(8), 0.2);
+    double gamma = mio::DistributionAdapter<std::lognormal_distribution<double>>::get_instance()(rng, std::log(16), 0.2);
```

Comment:
Please use meaningful names for variables. If you are referring to e.g. a paper, cite it and mention the symbols used in a comment above/behind the variable.

It would also be helpful to document parameter value choices, either here or preferably in the documentation of the parameters in parameters.h.

Reply:
What is the desired way to cite a paper? URL to the publication?

Reply:
Yes, preferably as a doi. For example [cpp/simulations/README.md](https://github.com/SciCompMod/memilio/blob/main/cpp/simulations/README.md?plain=1#L4) or with a text link [cpp/models/ode_seair/README.md](https://github.com/SciCompMod/memilio/blob/main/cpp/models/ode_seair/README.md?plain=1#L15)